src/prometheus_alert_rules/juju_machines_unstable.rule

```diff
@@ -0,0 +1,8 @@
+alert: JujuMachinesUnstable
+expr: abs(sum(juju_machine_state) - sum(juju_machine_state offset 3m)) >= bool 0.33 * sum(juju_machine_state)
```

Comment:
Why using 3min here in the expression? It seems to be chosen arbitrarily. The default prometheus scrape_interval is 1min, so maybe we should comply here (and keep the for interval).

Reply:
I changed it to `1m`